packages/@divvi/mobile/src/images/WelcomeLogo.tsx

```diff
@@ -9,11 +10,16 @@ interface Props {
 }
 
 export default function WelcomeLogo({ width = 243, height = 64, style }: Props) {
+  const CustomWelcomeLogo = getAppConfig().themes?.default?.assets?.welcomeLogo
+  if (CustomWelcomeLogo) {
+    return <CustomWelcomeLogo width={width} height={height} style={style} />
```

Comment:
I think we can display whatever was passed. Otherwise we'd need to document these.
```suggestion
    return <CustomWelcomeLogo />
```